Mage.Tests/src/test/java/org/mage/test/cards/single/clb/GaleWaterdeepProdigyTest.java

```diff
@@ -0,0 +1,49 @@
+package org.mage.test.cards.single.clb;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+/**
+ *
+ * @author R
+ * jayz
+ */
+public class GaleWaterdeepProdigyTest extends CardTestPlayerBase {
+
+    @Test
+    public void TestGaleWaterDeepProdigy() {
```

Comment:
What is going on with this test?

Reply:
I have modified the test to be more explicit as to what I want to test, and added comments to help explain it better.